src/main/java/org/jabref/model/openoffice/uno/UnoCast.java

```diff
@@ -0,0 +1,24 @@
+package org.jabref.model.openoffice.uno;
+
+import java.util.Optional;
+
+import com.sun.star.uno.UnoRuntime;
+
+public class UnoCast {
+
+    private UnoCast() { }
+
+    /**
+     * unoQI : short for UnoRuntime.queryInterface
+     *
+     * @return A reference to the requested UNO interface type if available,
+     *         otherwise null
+     */
+    public static <T> T unoQI(Class<T> zInterface, Object object) {
+        return UnoRuntime.queryInterface(zInterface, object);
+    }
+
+    public static <T> Optional<T> optUnoQI(Class<T> zInterface, Object object) {
```

Comment:
I don't understand why you need both methods

Reply:
unoQI was born to let me see the varying part.

```
XMultiServiceFactory msf = UnoRuntime.queryInterface(XMultiServiceFactory.class, doc);
XMultiServiceFactory msf = unoQI(XMultiServiceFactory.class, doc);
```

When breaking up the UNO-related collection, it became:

```
XMultiServiceFactory msf = UnoCast.unoQI(XMultiServiceFactory.class, doc);
```

At the time I either did not know about static imports,
or was not sure if it is accepted in JabRef code.

`import static` appears in the codebase, mostly in tests, occasionally elsewhere too.
But it does not tell me if it is in the "we are sorry, did not have time to remove"
or in the "yes, we love that category". 

Question: is `import static` allowed in JabRef? This would allow me to drop
the `UnoCast.` prefixes.

`optUnoQI` was born when I adjusted to a request: `do not return null`

It allows for example
```
return UnoCast.optUnoQI(XPropertySet.class, propertyContainer);
```
instead of
```
return Optional.ofNullable(UnoCast.unoQI(XPropertySet.class, propertyContainer));
```
or
```
return Optional.ofNullable(UnoRuntime.queryInterface(XPropertySet.class, propertyContainer));
```





It is not used in as many locations as I expected: in some cases
the cast is known (or usually expected) to be successful (the input is documented
to support the interface), in other cases internally I kept the testing for null
instead of converting to Optional then asking for isPresent

- UnoCast.unoQI : 40 matches
- UnoCast.optUnoQI : 6 matches

Reply:
In tests import Static is okay, but otherwise it shouldn't be really used.
If it's not too much effort, I would stick with the method that returns Optional so optUnoQI

Reply:
Dropped unoQI, renamed optUnoQI to cast